src/examples/rest/database.sql

```diff
@@ -44,7 +44,7 @@ CREATE TABLE api_key (
   api_key VARCHAR(255) NOT NULL UNIQUE,
   secret VARCHAR(255) NOT NULL,
   revoked BOOLEAN NOT NULL DEFAULT false,
-  roles JSON NOT NULL DEFAULT (JSON_ARRAY())
+  roles TEXT NOT NULL
```

Comment:
Is this change necessary? I would expect our docs and examples to break if we changed what kind of thing roles are. @steve-keep

Reply:
MySQL does not have array support so there are two options TEXT or JSON.

I prefer JSON as you know that it is validated when saved to the DB but it is not perfect.

@siowhoi I assume you had to change this because the JSON type was not compatible with the @Enum decorator?

Reply:
Yes, I get error: Invalid JSON text. To fix this error, I will need a customType to convert the values to valid JSON string.